tooling/ast_fuzzer/src/program/func.rs

```diff
@@ -715,6 +718,14 @@ impl<'a> FunctionContext<'a> {
         }
 
         if self.unconstrained() {
+            // For now only try prints in unconstrained code, were we don't need to create a proxy.
```

Comment:
R.e. `For now`: does this need a follow up?

Reply:
Yes, thanks for reminding, created https://github.com/noir-lang/noir/issues/8605